### DISCUSSION TITLE
Beatmap limitation options for multiplayer room

### DISCUSSION THREAD

**Discussion by LmanTW** [ppy/osu#27838] [Polls]
Add limitation options such as stars, length, ranked, etc to multiplayer rooms. This could improve the experience of those tho want to play maps that has a a specific difficulties / properties. This can also make organizing a room easier, as some people ignore the room title where most people write the requirements for the maps.

**Comment by CuredDDN** [comment]
I agree, this would help separate players who like playing lower ranked maps and higher ranked maps.

**Comment by Vergenter** [comment]
Perhaps just the forced Beatmap search filters should be used?
https://osu.ppy.sh/wiki/en/Beatmap_search

**Comment by ItsCollector** [comment]
I voted no but this is specific to osu!mania. The star rating gets really funky with certain types of charts, particularly long note maps where they are either under-starred or over-starred when comparing actual difficulty when playing those maps for myself. If there was adjustments to make star rating more accurate then I would click yes.

**Comment by Givikap120** [comment]
Tbh this is not really a poll because just moving a feature of stable bots to lazer
